source/features/link-to-github-io.tsx

```diff
@@ -30,15 +30,16 @@ function initRepo(signal: AbortSignal): void {
 }
 
 function initRepoList(signal: AbortSignal): void {
-	observe('a[href$=".github.io"][itemprop="name codeRepository"]', addRepoListLink, {signal});
+	// Also consider any old GitHub Pages repo like: resume/resume.github.com , issue: https://github.com/refined-github/refined-github/issues/6228
+	observe(':is(a[href$=".github.io"], a[href$=".github.com"])[itemprop="name codeRepository"]', addRepoListLink, {signal});
 }
 
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.hasRepoHeader,
 	],
 	asLongAs: [
-		() => Boolean(getRepo()?.name.endsWith('.github.io')),
+		() => Boolean(getRepo()?.name.endsWith('.github.io')) || Boolean(getRepo()?.name.endsWith('.github.com')),
```

Comment:
Can we avoid the duplicate function call here please?

Reply:
@FloEdelmann  i have updated it like this `Boolean(getRepo()?.name.endsWith('.github.io') || getRepo()?.name.endsWith('.github.com'))`. But not sure if this is what you were expecting.

Reply:
Sorry for being unclear. I meant we should avoid calling `getRepo()` twice here.

Reply:
Thank you  @fregante  for making the changes, I wasn't able to solve this one. @FloEdelmann  again thanks for suggesting this 🙂